src/components/Transaction.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { useEffect } from 'react';
+import { useState } from 'react'
+import { getAllCheckingAccounts, transaction } from '../api';
+
+export const Transaction = (props) => {
+
+
+    const [accounts,setAccounts] = useState([]); 
+    const [loading,setLoading] = useState(true); 
+    const [transactionValues,setTransactionValues] = useState({
+        amount:'',
+        currentAccount:''
+    }); 
+    const [error,setError] = useState(false); 
+    const [success,setSuccess] = useState(false); 
+
+    useEffect(()=>{
+        getAllCheckingAccounts()
+        .then(response =>{
+            setAccounts(response.data);
+            setLoading(false);  
+        },[]); 
+    })
+
+    const handleInputChange = ({target})=>{
+        
+        setTransactionValues({
+            ...transactionValues,
+            [target.name]:target.value
+        }) 
+    }
+
+    const handleSubmit = async(e) =>{
+        e.preventDefault(); 
+        try{
+            await transaction(props.type,transactionValues.amount,props.type,transactionValues.currentAccount);
+            setSuccess(true); 
+            setTimeout(() => {
+                setSuccess(false); 
+            }, 3000);
+
+        }catch(err){
+            setError(true);
+            console.log(err);  
+        }
+        
+        
+        
+
+    }
```

Comment:
Format this file so that you get rid of these extra spaces ;)

Reply:
I will.